src/configurations/crc-researcher/handleParticipantWorkflowChange.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import { PartialRow, TableUpdateTransactionRequest } from 'synapse-react-client/dist/utils/synapseTypes/Table/TableUpdate'
+import { Row } from 'synapse-react-client/dist/utils/synapseTypes'
+import { SynapseClient } from 'synapse-react-client'
+
+const handleParticipantWorkflowChange = async (event:any, newWorkflowState:string) => {
```

Comment:
this event:any is not ideal, should really use a type here but I see that we forgot to export the CustomControlCallbackData in SRC, shoot.

Reply:
You could duplicate the type in this file
CustomControlCallbackData = {
  data: QueryResultBundle | undefined
  selectedRowIndices: number[] | undefined
  refresh: () => void
}
typescript will see the typing of the event for the function and cross reference when you use it in routeConfig